python/py-icalendar/Portfile

```diff
@@ -24,9 +24,9 @@ long_description    The inspiration has come from the email package in \
 
 homepage            https://icalendar.readthedocs.org/
 
-checksums           rmd160  ed40725872ffd2517e678e2034edeb57a48ac0c0 \
-                    sha256  43c2db8632959d634f4e48f6e6131e706bf2cdddad488cf0b72fda079b796bad \
-                    size    142307
+checksums           rmd160  aa2ba0ae8d8b2e8b0ebf91178fffb5862cc454b8 \
+                    sha256  a697ce7b678072941e519f2745704fc29d78ef92a2dc53d9108ba6a04aeba466 \
+                    size    177169
 
 python.versions     312 313
 python.pep517_backend   hatch
```

Comment:
not your change but... the "py-hatch-vcs" should be a "build" dependency instead of "lib". Would you mind correcting that as part of this PR?